src/pkgmt/versioneer.py

```diff
@@ -139,7 +153,7 @@ def version(
     )
 
     # TODO: make it compatible with VersionerNonSetup
-    if not version_package and changelog_md_exists:
+    if not version_file and changelog_md_exists:
```

Comment:
let's get rid of this if statement. the line should be executed regardless of which type of project we're using